crates/python/qcs_sdk/__init__.pyi

```diff
@@ -1,29 +1,357 @@
-from ._execution_data import (
-    # classes
-    ExecutionData as ExecutionData,
-    ResultData as ResultData,
-    RegisterMap as RegisterMap,
-    RegisterMatrix as RegisterMatrix,
-    # errors
-    RegisterMatrixConversionError as RegisterMatrixConversionError,
-)
-
-from ._executable import (
-    # classes
-    Executable as Executable,
-    ExeParameter as ExeParameter,
-    JobHandle as JobHandle,
-    Service as Service,
-    # errors
-    ExecutionError as ExecutionError,
-)
-
-from ._register_data import (
-    # classes
-    RegisterData as RegisterData,
-)
-
-from .client import (
-    # classes
-    QCSClient as QCSClient,
-)
+import datetime
+from enum import Enum
+from typing import Dict, List, Sequence, Optional, Union, final
+
+import numpy as np
+from numpy.typing import NDArray
+
+from qcs_sdk.qpu import QPUResultData
+from qcs_sdk.qvm import QVMResultData
+from qcs_sdk.compiler.quilc import CompilerOpts
+
+from qcs_sdk.client import QCSClient as QCSClient
+
+class ExecutionError(RuntimeError):
+    """Error encountered when executing a program."""
+
+    ...
+
+@final
+class Executable:
+    """
+    The builder interface for executing Quil programs on QVMs and QPUs.
+    """
+
+    def __new__(
+        cls,
```

Comment:
Omitting the type here is fine, right? (referencing other comment in this file)

Reply:
Yeah, omitting the type is fine and what I normally do, not sure why I specified it on the other one. Removed it.